lib/time/testdata/test.star

```diff
@@ -0,0 +1,37 @@
+load('time', 'time')
+load('assert.star', 'assert')
+
+assert.eq(time.parse_time("2011-04-22T13:33:48Z"), time.parse_time("2011-04-22T13:33:48Z"))
+assert.eq(time.zero, time.parse_time("0001-01-01T00:00:00Z"))
+assert.true(time.parse_time("2010-04-22T13:33:48Z") < time.parse_time("2011-04-22T13:33:48Z"))
+assert.true(time.parse_time("2011-04-22T13:33:48Z") == time.parse_time("2011-04-22T13:33:48Z"))
+assert.true(time.parse_time("2012-04-22T13:33:48Z") > time.parse_time("2011-04-22T13:33:48Z"))
+
+assert.eq(time.parse_time("2020-06-26T17:38:36Z"), time.from_timestamp(1593193116))
+
+# zero
+assert.eq(time.zero.format("Mon Jan 2 15:04:05 -0700 MST 2006"), "Mon Jan 1 00:00:00 +0000 UTC 0001")
```

Comment:
Do we need time.zero? It makes sense in Go where a Time struct may be initialized, but that's not the case in Starlark. I would prefer that we eliminate it, and avoid having to reveal anything about the representation.